cypress/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
I guess you should not target es5. Maybe es2017 would be good.

Reply:
Done!